azkaban-common/src/test/java/azkaban/executor/MockExecutorLoader.java

```diff
@@ -49,7 +54,8 @@
   @Override
   public void uploadExecutableFlow(ExecutableFlow flow)
       throws ExecutorManagerException {
-    flows.put(flow.getExecutionId(), flow);
+    ExecutableFlow exFlow = ExecutableFlow.createExecutableFlowFromObject(flow.toObject());
```

Comment:
Why making a copy here is not obvious without reading this change history. Could you add a comment?